assets/css/project/auth.less

```diff
@@ -0,0 +1,31 @@
+.sjs-login {
```

Comment:
Please re-check styles with bootstrap linked to page. I didn't checked, but you might need some reset.